|  [About](./) | [Experiences](./work-exp.md) | [Education](./education.html) | [Publications](./scientific-publications.html) |  [Recipe](./recipe.html)| 


## Short bio

**Since June 2024:**
AIStrosight Engineer responsible for developing a Python library for longitudinal data analysis in medical research at APHP.<br>
Designed and developed a flexible, scalable library for building metrics and performing clustering on longitudinal data.

**2022-2024: **
CNRS Engineer| employed by Lesaffre's Data Science Team. <br>
@ [Partnership Lesaffre's Data Science/LBMC](https://www.rhone-auvergne.cnrs.fr/fr/cnrsinfo/accelerer-linnovation-dans-le-domaine-de-la-fermentation) <br>
  * Missions DataScience Lesaffre (60%) :
Development of a haploid strain detection algorithm using random forest with NGS data. Development of Nextflow pipelines (Metagenomics, Transcriptomics) designed to be launched from custom API and connected to Cloud storage. 
NGS data mining and modeling.
 * Mission LBMC (40%) : 
Development of a statistical framework for simulating and analyzing high-throughput RNAseq data with fixed, mixed, and interaction effects.

## Dev-ventures

**Longitudinal Data Analysis Library | AIStrosight, APHP** <br>
@[AIStrosight](https://team.inria.fr/aistrosight/) - (2024) <br>
Developed a flexible and scalable Python library designed to facilitate the creation of metrics and clustering on longitudinal medical data. This tool aims to enhance the analysis of temporal patterns in patient data, enabling advanced insights into health trends over time.


**High throughput RNAseq model fit | LBMC** <br>
@ [HTRfit](https://htrfit-lbmc-yvertlab-vortex-plasticity-mutation-477701eb488dfd9.gitbiopages.ens-lyon.fr) - _(2022 - 2024)_ <br>
This development project is the most advanced one I have participated in thus far! HTRfit is a statistical framework developed to investigate essential experimental parameters that influence your ability to detect expression changes, such as sequencing depth and the number of replicates. Moreover, by enabling the inclusion of fixed effects, mixed effects, and interactions in your RNA-seq data analysis, HTRfit provides the flexibility needed to effectively conduct your differential expression analysis. HTRfit is particularly well-suited for the analysis of a large number of samples or highly multiplexed experiments.<br>

**Shiny dashboard | BF2i** <br>
@ [APTA](https://github.com/aduvermy/shiny.git) - _(Oct 2020 - Nov 2020)_ <br>
Shiny application of BF2I allowing the visualization of RNA-seq data of the *pea aphid Acyrthosiphon pisum*. Allows performing PCA, clustering, heatmap, and time series visualization.



