|  [About](./) | [Experiences](./work-exp.md) | [Education](./education.html) | [Publications](./scientific-publications.html) |  [Recipe](./recipe.html)| 


## Short bio

I earned a master’s degree in bioinformatics in 2021. Shortly after, I started a two-year contract as a CNRS Engineer working in a research and development department at Lesaffre (@[Partnership Lesaffre's Data Science/LBMC](https://www.rhone-auvergne.cnrs.fr/fr/cnrsinfo/accelerer-linnovation-dans-le-domaine-de-la-fermentation)). There, I focused on developing detection algorithms and data processing pipelines, applying my expertise in statistics and machine learning to tackle complex data challenges in genomics and metagenomics.

Seeking new challenges and opportunities to advance method development, I joined AIStrosight in June 2024. Here, I’m dedicated to creating a Python library for analyzing longitudinal medical data, designed to integrate seamlessly with machine learning tools for enhanced predictive analysis.

As my contract with AIStrosight is nearing its end, I am currently seeking new challenges in developing innovative methods and technologies.

## Dev-ventures

**Longitudinal Data Analysis Library | AIStrosight, APHP** <br>
@[AIStrosight](https://team.inria.fr/aistrosight/) - (2024) <br>
Developed a flexible and scalable Python library designed to facilitate the creation of metrics and clustering on longitudinal medical data. 


**High throughput RNAseq model fit | LBMC** <br>
@ [HTRfit](https://htrfit-lbmc-yvertlab-vortex-plasticity-mutation-477701eb488dfd9.gitbiopages.ens-lyon.fr) - _(2022 - 2024)_ <br>
This development project is the most advanced one I have participated in thus far! HTRfit is a statistical framework developed to investigate essential experimental parameters that influence your ability to detect expression changes, such as sequencing depth and the number of replicates. Moreover, by enabling the inclusion of fixed effects, mixed effects, and interactions in your RNA-seq data analysis, HTRfit provides the flexibility needed to effectively conduct your differential expression analysis. HTRfit is particularly well-suited for the analysis of a large number of samples or highly multiplexed experiments.<br>

**Shiny dashboard | BF2i** <br>
@ [APTA](https://github.com/aduvermy/shiny.git) - _(Oct 2020 - Nov 2020)_ <br>
Shiny application of BF2I allowing the visualization of RNA-seq data of the *pea aphid Acyrthosiphon pisum*. Allows performing PCA, clustering, heatmap, and time series visualization.



