|  Home   | [Scientific publications](./scientific-publications.html) | Price |
| :-----: | :-------------------------------------------------------: | :----: |


[Scientific publications](./scientific-publications.html)

Software developer & Data Scientist in Biology <br>

## Short bio

Since 2022: CNRS Ingeneer employed by Lesaffre's Data Science Team <br>
@ [Partnership Lesaffre's Data Science/LBMC](https://www.rhone-auvergne.cnrs.fr/fr/cnrsinfo/accelerer-linnovation-dans-le-domaine-de-la-fermentation) <br>
  * Missions DataScience Lesaffre (60%) :
Development of a haploid strain detection algorithm using random forest with NGS data. Development of Nextflow pipelines (Metagenomics, Transcriptomics) designed to be launched from custom API and connected to Cloud storage. 
NGS data mining and modeling.
 * Mission LBMC (40%) : 
Development of a statistical framework for simulating and analyzing high-throughput RNAseq data with fixed, mixed, and interaction effects.

## Dev-ventures

**High throughput RNAseq model fit | LBMC** <br>
@ [HTRfit](https://htrfit-lbmc-yvertlab-vortex-plasticity-mutation-477701eb488dfd9.gitbiopages.ens-lyon.fr) - _(2022 - 2024)_ <br>
This development project is the most advanced one I have participated in thus far! HTRfit is a statistical framework developed to investigate essential experimental parameters that influence your ability to detect expression changes, such as sequencing depth and the number of replicates. Moreover, by enabling the inclusion of fixed effects, mixed effects, and interactions in your RNA-seq data analysis, HTRfit provides the flexibility needed to effectively conduct your differential expression analysis. HTRfit is particularly well-suited for the analysis of a large number of samples or highly multiplexed experiments.<br>

**Shiny dashboard | BF2i** <br>
@ [APTA](https://github.com/aduvermy/shiny.git) - _(Oct 2020 - Nov 2020)_ <br>
Shiny application of BF2I allowing the visualization of RNA-seq data of the *pea aphid Acyrthosiphon pisum*. Allows performing PCA, clustering, heatmap, and time series visualization.

## Trainings 

**Formation Introduction au Deep Learning | FIDLE CNRS** - _(2022 - 2023)_ <br>
Keras, Tensorflow, fine-tuning, CNN, transformers, encoder 

**Network statistical analysis | CNRS** - _(November 2023)_ <br>
Manipulating, visualizing, and sampling networks, statistical description, random graph models, and community detection (spectral clustering). 

## Scientific publications

* Terrone S, Valat J, Fontrodona N, Giraud G, Claude JB, Combe E, Lapendry A, Polv√®che H, Ameur LB, **Duvermy A**, Modolo L, Bernard P, Mortreux F, Auboeuf D, Bourgeois CF. RNA helicase-dependent gene looping impacts messenger RNA processing. Nucleic Acids Res. 2022 Sep 9;50(16):9226-9246. doi: 10.1093/nar/gkac717. PMID: 36039747; PMCID: PMC9458439.

* Rieux, A., Campos, P., **Duvermy, A.** et al. Contribution of historical herbarium small RNAs to the reconstruction of a cassava mosaic geminivirus evolutionary history. Sci Rep 11, 21280 (2021). https://doi.org/10.1038/s41598-021-00518-w

